app/create.js

```diff
@@ -17,74 +17,113 @@ define(['app/config', 'app/database', 'jquery', 'bootstrap', 'parsley', 'app/sel
 			numberOfFormItems--;
 		}
 	}
-  
-	// Function for adding elements to the form
-	function add_element() {
-		var newElement = $('#item-layout').clone(true).appendTo("#items table").removeAttr("id");
-		newElement.html(giveId(newElement.html(), formItemId));
-    newElement.html(giveRequired(newElement.html()));
+
+	function add_element_to_form() {
+		var newElement = $('#items table').cloneLayout();
+		newElement.replaceClone(["i", "required"], [formItemId, "required"]);
+		// Remove when clicked on close
 		newElement.on("click", ".remove", function() {
 			remove_element($(this));
 		});
+		// When the TAB is pressed, add a new line
+		removeKeybinds("keydown");
+		newElement.find("input:last").on('keydown', function(e) {
+			if (isTab(e.keyCode)) {
+					add_element_to_form();
+			}
+		});
 		numberOfFormItems++;
 		formItemId++;
 	}
-  
-  // Auxiliary replace functions
-  function giveId(string, formItemId) {
-    return string.replace(/{i}/g, formItemId);
-  }
-  function giveRequired(string) {
-    return string.replace(/{required}/g, 'required=""');
-  }
-  
-  // Auxiliary form functions
+
+	function isTab(keyCode){
```

Comment:
You are inconsistent when naming functions, you switch between camel case and snake case.